Suppress "Failed to load environment files" error on service stop

wgwoods opened the pull request:
The upgrade process removes /system-upgrade/ when it starts, so
obviously the stop-post task won't be able to load the EnvironmentFile
from /system-upgrade/.dnf-system-upgrade.

Add the "-" so this isn't considered a failure, so the system shuts down
normally.

Resolves: RHBZ#1317215

keszybz commented:
Yeah, I though about this as a fix too. But I couldn't actually reproduce the error as reported. I created a unit file that uses EnvironmentFile and tried to start the unit, remove the file, do systmectl-daemon reload,etc, and I didn't see the message described in the bug.

This change cannot hurt though. It's fine to merge it.

Conan-Kudo commented:
From my point of view, it looks okay. I don't see any obvious flaw with the modification.

wgwoods committed:
Send DNF stdout/stderr to text console (#1317195)

We previously dropped this, because it was causing duplicate text on
systems with plymouth - see #13 - but the side-effect is that now
there's no output whatsoever on systems *without* plymouth.

Too much output is better than not enough. Restore the output from DNF
here, and we'll try to fix plymouth so it doesn't duplicate its output
to its "details" view.

This reverts commit dcc334939aa888ae8e15291a16982062ec446226.
Resolves: RHBZ#1317195

wgwoods committed:
Fix "Failed to load environment files" error on service stop

The upgrade process removes /system-upgrade/ when it starts, so
obviously the stop-post task won't be able to load the EnvironmentFile
from /system-upgrade/.dnf-system-upgrade.

Add the "-" so a missing file isn't considered failure, so the system
shuts down normally.

Resolves: RHBZ#1317215

wgwoods commented:
...Now with a fix for issue #53 ([RHBZ#1317195](https://bugzilla.redhat.com/show_bug.cgi?id=1317195))!
